material-overrides/assets/stylesheets/kluster.css

```diff
@@ -537,7 +532,8 @@ li > nav.md-nav[data-md-level="4"] {
 }
 
 .md-typeset .tabbed-content {
```

Comment:
Ah shoot just noticed this is around all tabbed content, ideally we don't have this around everything, only things with text or lists, etc.